Sources/ProjectDescription/AnalyzeAction.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct AnalyzeAction: Equatable, Codable {
```

Comment:
```suggestion
public struct AnalyseAction: Equatable, Codable {
```
This should be Analyse to follow with Xcode's naming convention

Reply:
I think it's done properly and localized as it should :) ![Screenshot 2020-02-07 at 3 44 54 PM](https://user-images.githubusercontent.com/33914/74038641-d2f45d80-49c0-11ea-8151-4c838b309870.png)
As a point of reference at least in frameworks it's mostly US spelling e.g. `UIColor.gray`.

Reply:
👍🏼